app/views/flood_risk_engine/enrollments/_back_link.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= link_to(
       t("global.back"),
       enrollment_step_path(form.enrollment, form.enrollment.previous_step),
-      class: "back-link"
+      class: "link-back"
```

Comment:
Sorry my bad - I had added also broken this out as a partial in my branch but used `link-back` which is one of Polly's classes and adds the underline to the link. Will add `back-link ` back in also as both are required.